src/DependencyInjection/CsaGuzzleExtension.php

```diff
@@ -77,6 +81,18 @@ private function processLoggerConfiguration(array $config, ContainerBuilder $con
         }
     }
 
+    private function processMockConfiguration(array $config, ContainerBuilder $container, LoaderInterface $loader)
+    {
+        if (!$config['enabled']) {
+            return;
+        }
+
+        $container->setParameter('csa_guzzle.mock.storage_path', $config['storage_path']);
```

Comment:
You should inject the config values directly in the service. We do not need to expose two parameters, as they come from an explicit configuration.

This means that you should first load your service file, and then alter the definitions by injecting the two values.